docs/development/issues.md

```diff
@@ -0,0 +1,29 @@
+# GitHub Issue tracking cAdvisor
```

Comment:
We should include a note on assigning issues to contributors outside of the Maintainers list.
A simple comment might suffice.